generator/annotator.cpp

```diff
@@ -928,6 +928,21 @@ std::pair< std::string, std::string > Annotator::getReferenceAndTitle(clang::Nam
             } else {
                 mangle->mangleName(decl, s);
             }
+
+#ifdef _WIN32
+            s.flush();
+
+            const char* mangledName = cached.first.data();
+            if (mangledName[0] == 1) {
+                if(mangledName[1] == '_' || mangledName[1] == '?') {
+                    if(mangledName[2] == '?') {
+                        cached.first = cached.first.substr(3);
+                    } else {
+                        cached.first = cached.first.substr(2);
+                    }
+                }
```

Comment:
Does it always start with a '?' or a '_'

Reply:
So removing the \1, for sure we shoud.
But maybe we shoud keep the symbol as is (with ? and _).  what do you think?

Reply:
Hm - ? is also invalid in filenames on Windows. 
Then keeping only the _ (but stripping the ? afterwards) makes the code even less readable for no real benefit imho. If the actual identifier has an underscore prefix it will be *after* the question mark and not before. So I think we have no ambuigity here and it does not matter.

Reply:
ah ok, i forgot ? was not allowed